solidity-v1/dashboard/src/components/modal/staking/undelegate/components/InfoList.jsx

```diff
@@ -15,6 +17,14 @@ export const InfoList = ({ undelegationPeriod, undelegatedAt }) => {
           Tokens will be available in {undelegationPeriodInRelativeTime}, on{" "}
           {undelegationCompletedAt.format("D MMM YYYY")}.
           {/* TODO: Add `add to calendar` button */}
+          <AddToCalendar
+            {...UNDELEGATE_STAKE_CALENDAR_EVENT}
+            startsAt={undelegationCompletedAt.format("YYYY-MM-DD HH:mm:ss")}
+            endsAt={undelegationCompletedAt
+              .add(15, "minutes")
+              .format("YYYY-MM-DD HH:mm:ss")}
```

Comment:
Yeah, I just gave it 15 minutes because I think we should specify the `endDate` to avoid the situation where this event takes place in calendar until the end of the world 😅